templates/problem/voting-controls.html

```diff
@@ -0,0 +1,65 @@
+<hr style="padding-top: 0.1em">
+
+<div class="vote-form" id="id_vote_form_box">
+    <div id="id_has_voted_prompt">
+        <span class="vote-form-info">{{ _('Current Vote:') }}</span>
+        <span id="id_current_vote_value" class="vote-form-value"></span>
+        <br/>
+        <br/>
+        <span class="vote-form-text">{{ _('Change your vote below!') }}</span>
+        <br/>
+        <br/>
+    </div>
+    <div id="id_has_not_voted_prompt">
+        <span class="vote-form-text">{{ _("Cast your vote on this problem's weighting below!") }}</span>
+        <br/>
+        <br/>
+    </div>
+    <form id="id_delete_vote_form">
+        {% csrf_token %}
+    </form>
+    <form id="id_vote_form">
+        {% csrf_token %}
+        <span class="vote-form-info">{{ _('Points') }}</span>
+        <input id="id_points" class="vote-form-value" type="number" step="1"
+               min="{{ min_possible_vote }}" max="{{ max_possible_vote }}" name="points">
+        <br/>
```

Comment:
Isn't there usually a space before the `/`? I think you should probably fix this everywhere it appears, unless (and check this), the codebase disagrees with me. Please make a note (on this comment) either way.

Reply:
The rest of the codebase seems to use `<br>` with no slash. Should we use that instead?

Reply:
Follow the codebase, yes.